lib/defaults.go

```diff
@@ -27,9 +23,12 @@ func GetDefaultBin() string {
 const (
 	DefaultMirror             = "https://releases.hashicorp.com/terraform"
 	DefaultLatest             = ""
+	distributionTerraform     = "terraform"
+	distributionOpenTofu      = "opentofu"
 	installFile               = "terraform"
 	InstallDir                = ".terraform.versions"
+	pubKeySuffix              = ".asc"
 	recentFile                = "RECENT"
+	TerraformPrefix           = "terraform_"
 	tfDarwinArm64StartVersion = "1.0.2"
-	VersionPrefix             = "terraform_"
```

Comment:
This was a public constant, we maybe we need to consider if it was being used externally